nebula/ui/components/MZStepNavigation.qml

```diff
@@ -116,4 +116,9 @@ ColumnLayout {
             implicitHeight: flickable.height
         }
     }
+
+    //Fix for VPN-5663
+    Item {
+        Layout.fillHeight: true
+    }
```

Comment:
I honestly am not entirely sure why this resolves the issue 😅

Reply:
I'm curious how you even thought to try this.

Reply:
You make a lot of desperate reaches when you don't have access to the inspector on mobile. I was really just throwing stuff at the wall to try to figure out what component was causing the blank spaces... and I still don't know